components/contextualhelp/stories/contextualhelp.stories.js

```diff
@@ -0,0 +1,76 @@
+// Import the component markup template
+import { Template } from "./template";
+import { html } from "lit-html";
+
+import { default as ActionButtonStories } from "@spectrum-css/actionbutton/stories/actionbutton.stories.js";
+
+// More on default export: https://storybook.js.org/docs/web-components/writing-stories/introduction#default-export
+export default {
+  title: "Contextual Help",
+  description: "The Contextual Help component is...",
+  component: "ContextualHelp",
+  argTypes: {
+    title: {
+      name: "Title",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Content",
+      },
+    },
+    body: {
+      name: "Body",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Content",
+      },
+    },
+    iconName: {
+      name: "Icon",
+      type: { name: "string", required: true },
+      defaultValue: "Info",
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+        defaultValue: { summary: "Info" }
+      },
+      options: ["Info", "Help"],
+      control: "select"
+    },
+    popoverPlacement: { table: { disable: true }},
```

Comment:
```suggestion
    popoverPlacement: { table: { disable: true }},
```

Would it be valuable to import and make the popover placement available for customization in storybook?

Reply:
I can. The issue is just that there's inline styling and the popover needs to come before the button in the DOM in order for it to be above the button. I can of course add in conditionals for all of that but I didn't know if that would muddy things up

Reply:
I think I got it pretty much working! Let me know what you think